featuretools/primitives/standard/transform_primitive.py

```diff
@@ -215,7 +215,7 @@ class Week(TransformPrimitive):
         """
     name = "week"
     input_types = [ColumnSchema(logical_type=Datetime)]
-    return_type = ColumnSchema(logical_type=Ordinal(order=range(1, 53)), semantic_tags={'category'})
+    return_type = ColumnSchema(logical_type=Ordinal(order=list(range(1, 53))), semantic_tags={'category'})
```

Comment:
Should this be `range(1, 54)` here?  I think some years can technically have 53 weeks depending on how the calendar falls.